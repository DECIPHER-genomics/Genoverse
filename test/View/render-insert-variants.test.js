const { afterTest, testTrackRenderStatic } = require('../utils');

describe('Correctly render insert variants:', () => {
  afterEach(afterTest);

  const track = { _testClass: Genoverse.Track.dbSNP, url: false, margin: 0, featureMargin: {} };

  it('when scale > 1', () => testTrackRenderStatic([{ start: 10, end: 5 }], track));

  it('when scale < 1', () => testTrackRenderStatic([{ start: 100, end: 50 }], track, { end: 1000 }));
});
