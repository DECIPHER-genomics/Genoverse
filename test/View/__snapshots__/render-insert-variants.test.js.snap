// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly render insert variants: when scale < 1 1`] = `"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGQAAAAMCAYAAACURRhjAAAABmJLR0QA/wD/AP+gvaeTAAAAcklEQVRYhe3TsQmAQBBE0c9hZmyitdiOkW1Yk7m1WIHpBZooYragsovOg80uGPgc2NRAZ3wrNyTvAXKlIMEoSDAKEoyCBKMgwShIMIXxXQ+Ubw4RuxbIwAI0zlt+rwJmYN1vwv6r5GEJGDljHDd4jvq6DcGyDEZlhwxAAAAAAElFTkSuQmCC"`;

exports[`Correctly render insert variants: when scale < 1 2`] = `"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGQAAAABCAYAAAAo2wu9AAAABmJLR0QA/wD/AP+gvaeTAAAADklEQVQYlWNgGAWDCgAAAZEAAejZ5ZUAAAAASUVORK5CYII="`;

exports[`Correctly render insert variants: when scale > 1 1`] = `"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGQAAAAMCAYAAACURRhjAAAABmJLR0QA/wD/AP+gvaeTAAAAZ0lEQVRYhe3XsQnAIBRF0UtaR7HJiNnEqbJMGlOYIA4g74vvwAfLB7cS1nMBVT1ilkM9wEYOEoyDBOMgwThIMA4SjIMEs2KQUz3Augw8tI9hFm/ZXgJuWoz6vZN00eYKPcZ/RbpogheofA0HZ+9G9AAAAABJRU5ErkJggg=="`;

exports[`Correctly render insert variants: when scale > 1 2`] = `"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGQAAAABCAYAAAAo2wu9AAAABmJLR0QA/wD/AP+gvaeTAAAADklEQVQYlWNgGAWDCgAAAZEAAejZ5ZUAAAAASUVORK5CYII="`;
